test/performance/benchmarks_test.go

```diff
@@ -0,0 +1,152 @@
+// +build performance
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package performance
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	pkgTest "github.com/knative/pkg/test"
+	"github.com/knative/pkg/test/ingress"
+	"github.com/knative/serving/test"
+	"github.com/knative/test-infra/shared/junit"
+	"github.com/knative/test-infra/shared/loadgenerator"
+	"github.com/knative/test-infra/shared/testgrid"
+)
+
+const (
+	reqTimeout    = 30 * time.Second
+	app = "helloworld"
+)
+
+var loads = [...]int32{1, 100}
+
+func filename(name string) string {
+	// Replace characters in `name` with characters for a file name.
+	replacer := strings.NewReplacer("/", "_")
```

Comment:
just https://golang.org/pkg/strings/#ReplaceAll and method probably is not needed.

Reply:
Done.  I thought I'd need to replace more than "/".